### PR TITLE
fixed

### PR DESCRIPTION
**Pull request by OmerLiberman**

### CODE REVIEW

**Comment by yochze** [comment]
The PR contains removal of `.values` from all fit/predict

We added those to support non DF functions when serving a model

**Comment by OmerLiberman** [comment]
fixed this.

File: xgboost/SKTrainer.py [hunk 1]
```diff
@@ -14,41 +14,36 @@
...
 
 
 class SKTrainer:
-	DIGITS_TO_ROUND = 3
-
 	def __init__(self, model, train_set, test_set, output_model_name, testing_mode, folds=None):
 		self.__model = model
 		self.__x_train, self.__y_train = train_set
 		self.__x_test, self.__y_test = test_set
+		self.__all_data_concatenated = pd.concat([pd.concat([self.__x_train, self.__x_test], axis=0),
```
Comment (yochze) [inline xgboost/SKTrainer.py:28]: why this was added?

File: xgboost/SKTrainer.py [hunk 2]
```diff
@@ -14,41 +14,36 @@
 
 from cnvrg import Experiment
 from cnvrg.charts import Heatmap, Bar, Scatterplot
-from cnvrg.charts.pandas_analyzer import PandasAnalyzer, MatrixHeatmap
+from cnvrg.charts.pandas_analyzer import MatrixHeatmap
 
-from sklearn.model_selection import KFold
 from sklearn.model_selection import cross_validate
-from sklearn.metrics import classification_report, confusion_matrix, roc_curve, accuracy_score, mean_squared_error
+from sklearn.metrics import classification_report, confusion_matrix, roc_curve, accuracy_score, zero_one_loss
```
Comment (yochze) [inline xgboost/SKTrainer.py:20]: why this was changed?

**Pull request merged**